scripts/validate/fileheader

```diff
@@ -14,7 +14,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-set -eu -o
+set -eu -o pipefail
```

Comment:
We should probably check if it's already supported on all distros; this script uses a POSIX shell (not Bash), and  `pipefail` was not part of POSIX until recently; some distros (specifically; Debian) may not support it yet; also see

- https://github.com/koalaman/shellcheck/issues/2555

Reply:
Either way, `set -eu -o` without an argument to that `-o` is wrong, so this should be updated even in master (one way or the other) :sweat_smile:

Reply:
Ah! Yes! I blame ..... me for that (hehe and look at that diff 🙈🥹) https://github.com/docker/containerd-packaging/commit/93ae009441671c3e3f0488ae2890644211c99d8a